test_programs/execution_success/signed_truncation/Prover.toml

```diff
@@ -0,0 +1 @@
+x = "0x7fffffffffffffff"
```

Comment:
Was this change made to see a specific value or were there issues with i64::min?

Reply:
I just wanted to start off with a positive value. We could used `i64::min` for this